packages/react/src/auto/AutoForm.ts

```diff
@@ -50,16 +50,27 @@ export const useValidationResolver = (metadata: ActionMetadata | undefined) => {
 export const useFormFields = (
   metadata: ActionMetadata | undefined | null,
   options: { include?: string[]; exclude?: string[]; fields?: string[]; select?: Record<string, any> }
-): (readonly [string, FieldMetadata])[] => {
+): readonly { path: string; fieldMetadata: FieldMetadata }[] => {
```

Comment:
`field.fieldMetadata` seems a bit weird. How about `field.metadata`?

Reply:
Updated to just be `metadata`